airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1326,3 +1326,90 @@ definitions:
       $parameters:
         type: object
         additionalProperties: true
+interpolation:
+  variables:
+    - title: config
+      description: The connector configuration. The config's keys match the keys defined in the spec.
+    - title: headers
+      description: The HTTP headers from the last response received from the API.
+    - title: last_records
+      description: List of records extracted from the last response received from the API.
+    - title: record
+      description: The record being processed.
+    - title: response
+      description: The body of the last response received from the API.
+    - title: stream_interval
+      description:
+    - title: stream_partition
+      description: The current partition being processed.
+    - title: stream_slice
+      description: This variable is deprecated. Use stream_interval or stream_partition instead.
+    - title: stream_state
+      description: The current state of the stream.
+  macros:
+    - title: Now (Local)
+      description: Returns the current date and time using the local timezone.
```

Comment:
"local" will be with respect to the machine running the sync, right? In which case should this be not-recommended?

Reply:
good point. It doesn't seem like it's used in any existing connector so I vote we remove the macro entirely

Reply:
Makes sense to me!

Reply:
PR to remove the macro: https://github.com/airbytehq/airbyte/pull/25404